ibis/sql/postgres/tests/test_postgis.py

```diff
@@ -0,0 +1,211 @@
+import geopandas as gp
```

Comment:
This should be `gp = pytest.importorskip('geopandas')` to allow folks who don't have `geopandas` installed to run the test suite.